sale_order_line_case_number/__manifest__.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2023 Quartile Limited
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+{
+    "name": "Sale Order Line Case Number",
+    "category": "Sale",
+    "version": "15.0.1.0.0",
+    "author": "Quartile Limited",
+    "website": "https://www.quartile.co",
+    "license": "LGPL-3",
```

Comment:
```suggestion
    "license": "AGPL-3",
```
Please change the license to AGPL.  We should apply AGPL when no proprietary code is expected to depend on the module.